R/tm_g_distribution.R

```diff
@@ -648,7 +648,9 @@ srv_distribution <- function(id,
         )
       }
 
-      qenv
+      qenv %>%
+        # used to display table when running show-r-code code
+        teal.code::eval_code(quote(summary_table))
```

Comment:
It is my preference to incorporate the variable in the data creation expression for the purpose of printing, as follows:
```
qenv <- teal.code::eval_code(
          qenv,
          substitute(
            expr = {
              strata_vars <- strata_vars_raw
              summary_table <- ANL %>%
                dplyr::group_by_at(dplyr::vars(dplyr::any_of(strata_vars))) %>%
                dplyr::summarise(
                  min = round(min(dist_var_name, na.rm = TRUE), roundn),
                  median = round(stats::median(dist_var_name, na.rm = TRUE), roundn),
                  mean = round(mean(dist_var_name, na.rm = TRUE), roundn),
                  max = round(max(dist_var_name, na.rm = TRUE), roundn),
                  sd = round(stats::sd(dist_var_name, na.rm = TRUE), roundn),
                  count = dplyr::n()
                )
              summary_table
            },
            env = list(
              dist_var_name = dist_var_name,
              strata_vars_raw = c(g_var, s_var),
              roundn = roundn
            )
          )
        )
```

Reply:
The idea was to separate the printing from the logic. However, the general pattern follows your comment, I will change it! :+1:

Reply:
Done in 92ca6e5e